src/features/student/additionalSkills/components/interactions/inputs/AdditionalSkillCommentInput/AdditionalSkillCommentInput.test.ts

```diff
@@ -0,0 +1,216 @@
+      expect(input.props('required')).toBe(false)
+    })
+
+    BddTest().then('it should pass custom maxlength prop to AvInput', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('maxlength')).toBe(200)
+    })
+
+    BddTest().then('it should use custom label', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('label')).toBe('Custom Label')
+    })
+
+    BddTest().then('it should pass custom errorMessage to AvInput', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('errorMessage')).toBe('Custom error message')
+    })
+  })
+
+  BddTest().and('the user updates the input value', () => {
+    beforeEach(async () => {
+      vi.clearAllMocks()
+      wrapper = mount(AdditionalSkillCommentInput, {
+        props: {
+          modelValue: ''
+        },
+        global: { stubs }
+      })
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      await input.vm.$emit('update:modelValue', 'New comment value')
+      await wrapper.vm.$nextTick()
+    })
+
+    BddTest().then('it should emit update:modelValue event', () => {
+      expect(wrapper.emitted('update:modelValue')).toBeTruthy()
+      expect(wrapper.emitted('update:modelValue')?.[0]).toEqual(['New comment value'])
+    })
+  })
+
+  BddTest().and('the component receives a modelValue prop', () => {
+    beforeEach(() => {
+      vi.clearAllMocks()
+      wrapper = mount(AdditionalSkillCommentInput, {
+        props: {
+          modelValue: 'Initial comment'
+        },
+        global: { stubs }
+      })
+    })
+
+    BddTest().then('it should pass the modelValue to AvInput', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('modelValue')).toBe('Initial comment')
+    })
+  })
+
+  BddTest().and('the component is mounted with empty modelValue', () => {
+    beforeEach(() => {
+      vi.clearAllMocks()
+      wrapper = mount(AdditionalSkillCommentInput, {
+        props: {
+          modelValue: ''
+        },
+        global: { stubs }
+      })
+    })
+
+    BddTest().then('it should pass empty string to AvInput', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('modelValue')).toBe('')
+    })
+  })
+
+  BddTest().and('the component is mounted with error message', () => {
+    beforeEach(() => {
+      vi.clearAllMocks()
+      wrapper = mount(AdditionalSkillCommentInput, {
+        props: {
+          errorMessage: 'This field is required'
+        },
+        global: { stubs }
+      })
+    })
+
+    BddTest().then('it should display the error message', () => {
+      const input = wrapper.findComponent({ name: 'AvInput' })
+      expect(input.props('errorMessage')).toBe('This field is required')
+    })
+  })
+
+  BddTest().and('the component receives multiple updates', () => {
```

Comment:
```suggestion
  BddTest().when('the component receives multiple updates', () => {
```

S'il n'y a pas de when avant un then, alors ce and doit être un when